CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Fixed: `selector-max-specificity` no longer crashes on selectors containing `:not()` pseudo-classes.
 - Fixed: `time-no-imperceptible` correctly handles negative time.
 - Fixed: `function-calc-no-unspaced-operator` correctly interprets negative fractional numbers without leading zeros and those wrapped in parentheses.
+- Fixed: `number-max-precision` now better handle case insensitive `@import` at-rules.
```

Comment:
@jeddy3 before we ignore only lowercase, right `now ignore uppercase and mixed case`